ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -264,21 +265,21 @@ protocolInfoCardano
   -> Maybe PBftSignatureThreshold
   -> Byron.Update.ProtocolVersion
   -> Byron.Update.SoftwareVersion
-  -> Maybe ByronLeaderCredentials
+  -> [ByronLeaderCredentials]
```

Comment:
Oh, interesting, I didn't have the guts to do this as well : -)

..and the uniformity simplifies things a bit, indeed.